tests/checker/bad/gold/libchaincycle.scilla.gold

```diff
@@ -2,7 +2,7 @@
   "errors": [
     {
       "error_message":
-        "Cyclic dependence found when importing TestLibCycle1.",
+        "Cyclic dependence found when importing TestLibCycle1 (mapped to TestLibCycle1).",
```

Comment:
This repetition is a bit confusing: `TestLibCycle1 (mapped to TestLibCycle1)`

Reply:
So it's like that because there was no mapping for it and we used the library name itself. Do you think I better not print the "mapped to" part if they're same?

Reply:
There are multiple options (e.g. printing the full path of a conflicting library), but let's go with your suggestion.

Reply:
I've now fixed this. Please have a look.